Bow.podspec

```diff
@@ -0,0 +1,16 @@
+Pod::Spec.new do |s|
+  s.name        = "Bow"
+  s.version     = "0.0.0.1"
+  s.summary     = "Bow is a library for Typed Functional Programming in Swift."
+  s.homepage    = "https://github.com/arrow-kt/bow"
+  s.license     = { :type => "Copyright" }
+  s.authors     = { "Tomas Ruiz" => "truizlop@gmail.com" }
```

Comment:
This should be expressed int he same terms as Arrow. I'm sure @truizlop is sharing ownership of this project he started with everyone that contributes to it now that is hosted in the arrow org in the same way Arrow shares ownership with all its contributors and authors

Reply:
Done! :)
Shall we merge the PRs?